src/main/webapp/js/user-page-loader.js

```diff
@@ -82,6 +100,10 @@ function buildMessageDiv(message) {
   const bodyDiv = document.createElement('div');
   bodyDiv.classList.add('message-body');
   bodyDiv.innerHTML = message.text;
+  if (message.imageUrl) {
+    bodyDiv.innerHTML += '<br/>';
+    bodyDiv.innerHTML += '<img src="' + message.imageUrl + '" />';
```

Comment:
maybe consider just making this one '+=' statement, just so not as many retrievals of innerHTML are necessary (obviously won't improve performance, but probably just cleaner this way?

Reply:
Sounds good